README.md

```diff
@@ -43,6 +43,36 @@ yarn dev
 rails s
 ```
 
+## Postgres setup
```

Comment:
@BartkiewiczM Please take a look, this is what we did locally to make the app work with PG. But I believe you've been in touch with PG way more and may have some suggestions to simplify things.

Reply:
@BartkiewiczM Are you testing or planning to test in a near future? 
@franiakat maybe we could add it as it is - any changes for simplification of this process can be applied without much work, and they won't influence anything as it's only the readme :)

Reply:
I've realized that all the people that had mysql setup already switched to postgres. I think while setting up from scratch there will be no/different problems. I'll test a clean setup on a new machine this week and come back here :)